### DISCUSSION TITLE
Atlantis integration improvements

### DISCUSSION THREAD

**Discussion by alikhajeh1** [infracost/infracost#1179] [Ideas & Feature Requests]
# The issues in this discussion have been addressed in the [updated integration](https://github.com/infracost/infracost-atlantis/)

---

### Original content

Our [current Atlantis integration](https://github.com/infracost/atlantis-demo/pull/6#issuecomment-795889174) outputs the CLI diff text at the end of the Atlantis output. That means it's tucked-away at the bottom of the expandable "Show Output" section, which is easy to miss.

## Note for community
Please 👍 or comment on the ideas that resonate with you!

**Comment by alikhajeh1** [comment]
### Separate comment for cost estimates

We made [an example](https://github.com/infracost/infracost-atlantis/tree/master/examples) of how to post a GitHub PR comment for a Terraform mono-repo. If you're a GitHub user, can you please try this and let us know what you think?

**Reply by alikhajeh1** [reply]
See https://github.com/runatlantis/atlantis/issues/1954 for an option that makes this much simpler

**Comment by alikhajeh1** [comment]
### Native integration

Unfortunately Atlantis doesn't have a plugins concept. However, it's an open source project, so we could discuss the idea of native integration with the Atlantis maintainers. They recently added [integration with Conftest](https://www.runatlantis.io/docs/policy-checking.html).

Would you be interested in native Infracost integration?

**Reply by alikhajeh1** [reply]
See https://github.com/runatlantis/atlantis/discussions/1955 for another option

**Comment by alikhajeh1** [comment]
### Version the infracost-atlantis integration and setup corresponding Docker tags

https://github.com/infracost/infracost-atlantis/issues/10 has received a few upvotes already!

**Comment by alikhajeh1** [comment]
### Conftest integration

How could the Atlantis Conftest integration work with Infracost?

**Comment by david-heward-unmind** [comment]
Support fot github applications?

**Reply by alikhajeh1** [reply]
@davidh-unmind we'd like to do that too, we've been working on parsing HCL code directly makes a GH App simpler (so we can run Infracost against the main/master branch, then against the PR branch and compare, without users having to define cloud creds or TF secrets). See https://www.infracost.io/docs/guides/v0.10_migration/ for the beta, we plan to release v0.10 this week :)

**Reply by alikhajeh1** [reply]
@davidh-unmind we’re working on native integration with GitHub using [GitHub Apps or OAuth Apps](https://docs.github.com/en/developers/apps/getting-started-with-apps/differences-between-github-apps-and-oauth-apps). Do you have a preference on which you need? If so, why?
